src/main/java/com/playonlinux/framework/SetupWizard.java

```diff
@@ -103,6 +109,47 @@ public void execute(Message message) {
                 }
         );
     }
+
+    /**
+     * Show the content of a licence file
+     * @param textToShow a message above the licence
+     * @param licenceFile the licence file to display (with 'from java.io import File')
+     * @throws ScriptFailureException
+     * @throws CancelException
+     */
+    public void licence(String textToShow, File licenceFile) throws ScriptFailureException, CancelException {
```

Comment:
You need to define three methods: 
- licence(String textToShow, String licenceText)
- licenceFile(String textToShow, File licenceFile)
- licenceFile(String textToShow, String licenceFilePath)